client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts

```diff
@@ -281,7 +281,7 @@ function getFallbackDestination( {
 			if ( isJetpackCloud() ) {
 				if ( redirectCloudCheckoutToWpAdmin() && adminUrl ) {
 					debug( 'checkout is Jetpack Cloud, returning wp-admin url' );
-					return `${ adminUrl }admin.php?page=jetpack#/my-plan`;
+					return `${ adminUrl }admin.php?page=jetpack#/recommendations`;
```

Comment:
There's a suite of tests for this function in `client/my-sites/checkout/composite-checkout/test/composite-checkout-thank-you.js` that should be updated for this change.

Reply:
Thanks for adding the tests!